Decred Wallet/Features/Wallet Setup/WalletSetupViewController.swift

```diff
@@ -10,28 +10,10 @@ import Foundation
 import UIKit
 
 class WalletSetupViewController: WalletSetupBaseViewController {
-    @IBOutlet weak var infoText: UILabel!
-    @IBOutlet weak var restoreWallet: UILabel!
-    @IBOutlet weak var createWallet: UILabel!
-    @IBOutlet weak var build: UILabel!
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        createWallet.text = LocalizedStrings.createNewWallet
-        restoreWallet.text = LocalizedStrings.restoreExistingWallet
-        infoText.text = LocalizedStrings.createOrRecoverWallet
-    
-        let dateformater = DateFormatter()
-        dateformater.dateFormat = "yyyy-MM-dd"
-        let netType = BuildConfig.IsTestNet ? "testnet" : BuildConfig.NetType
-        build?.text = "build \(netType) " + dateformater.string(from: AppDelegate.compileDate)
-    }
-    
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        if segue.identifier == Segues.toNewWalletCreation.rawValue {
+        if segue.identifier == "toNewWalletCreation" {
```

Comment:
Any reason for comparing with string literals instead of the enum raw value??

Reply:
The values are not reused / used anywhere else other than here. Don't see a need for an enum.